api/src/main/java/org/openmrs/ConceptAnswer.java

```diff
@@ -251,4 +251,13 @@ public int compareTo(ConceptAnswer ca) {
 		}
 		return (getSortWeight() < ca.getSortWeight()) ? -1 : (getSortWeight() > ca.getSortWeight()) ? 1 : 0;
 	}
+	

```

Comment:
Actually the modification just shows a space which is well and introduced by the openmrs formatter